src/components/MAS/Mas.tsx

```diff
@@ -0,0 +1,137 @@
+                    navigate('/');
+                }
+            }).catch(error => {
+                console.warn(error);
+            });
+        }
+    }, []);
+
+    /* Class Names */
+    const classTabList = classNames({
+        'tabsList': true
+    });
+
+    const classTab = classNames({
+        'react-tabs__tab tab': true
+    });
+
+    return (
+        <div className="h-100 d-flex flex-column overflow-hidden">
+            <Header />
+
+            <Container className="flex-grow-1 py-5">
+                {MAS &&
+                    <div className="h-100 d-flex flex-column">
+                        <Row className="mb-2">
+                            <Col>
+                                <TitleBar title={MAS.name}
+                                    subTitle="Machine Annotation Services"
+                                />
+                            </Col>
+                            {KeycloakService.IsLoggedIn() &&
+                                <Col className="col-lg-auto">
+                                    <button type="button"
+                                        className="primaryButton px-3 py-1"
+                                        onClick={() => navigate(`edit`)}
+                                    >
+                                        Edit
+                                    </button>
+                                </Col>
+                            }
+                        </Row>
+                        <Row className="flex-grow-1">
+                            <Col lg={{ span: 4 }}>
+                                <IDCard identifier={MAS.id}
+                                    IDCardProperties={{
+                                        name: MAS.name,
+                                        containerImage: MAS.containerImage,
+                                        containerTag: MAS.containerTag,
+                                        topicName: MAS.topicName,
+                                        sourceCodeRepository: MAS.sourceCodeRepository,
+                                        codeMaintainer: MAS.codeMaintainer,
+                                        codeLicense: MAS.codeLicense,
+                                        supportContact: MAS.supportContact,
+                                        slaDocumentation: MAS.slaDocumentation,
+                                        maxReplicas: MAS.maxReplicas
+                                    }}
+                                />
+                            </Col>
+                            <Col lg={{ span: 8 }}>
+                                <Tabs className="h-100 d-flex flex-column">
+                                    <TabList className={classTabList}>
+                                        <Tab className={classTab} selectedClassName="active"> Service description </Tab>
+                                        <Tab className={classTab} selectedClassName="active"> Target Digital Object Filters </Tab>
```

Comment:
Oh yes this reminds me, the old filters didn't align with the new json-path structure. Did you have a look at this here? Otherwise I will create a new task for it.

Reply:
Already covered the loading of the filters, with removing the prefix ('$.') part. Also have to apply the prefix again when posting the MAS with the filters, but that will come in a new PR as previously discussed.